source/stylesheets/base/_buttons.scss

```diff
@@ -1,35 +1,36 @@
 $_button-background-color: $action-color;
 $_button-background-color-hover: shade($action-color, 20%);
 
-#{$all-buttons} {
+#{$all-buttons},
+.button {
+  -webkit-font-smoothing: antialiased;
   appearance: none;
   background-color: $_button-background-color;
   border: 0;
-  border-radius: $base-border-radius;
+  border-radius: $large-border-radius;
   color: contrast-switch($_button-background-color);
   cursor: pointer;
-  display: inline-block;
+  display: block;
   font-family: $base-font-family;
-  font-size: 16px;
-  -webkit-font-smoothing: antialiased;
-  font-weight: 600;
+  font-size: $base-font-size;
+  font-weight: 800;
   line-height: 1;
-  padding: $small-spacing $base-spacing;
+  padding: 9px $small-spacing;
```

Comment:
Maybe use ems instead of pixels if you're going to set the top and bottom padding explicitly. This way it will adapt based on the button's font size.